dev/README.md

```diff
@@ -0,0 +1,48 @@
+Overview
+========
+
+Welcome to Astronomer! This project was generated after you ran 'astro dev init' using the Astronomer CLI. This readme describes the contents of the project, as well as how to run Apache Airflow on your local machine.
```

Comment:
Since we are already committing the astro project, should we rephrase a bit?

It says "after you ran ....", but the developers are not actually running the command but is readily getting it checked into git.

Reply:
As discussed offline, I have removed this README.md to avoid confusion.